uni/lib/view/exams/widgets/exam_row.dart

```diff
@@ -54,11 +54,13 @@ class _ExamRowState extends State<ExamRow> {
                 children: <Widget>[
                   Column(
                     mainAxisAlignment: MainAxisAlignment.center,
-                    children: [
-                      ExamTime(
-                        begin: widget.exam.beginTime,
-                      ),
-                    ],
+                    children: widget.exam.beginTime != '00:00'
```

Comment:
It seems a bit weird to use the formatted string to assess prop existence. Consider removing the formatted getters from the `Exam` model, and use the date times here. Have the widgets format the dates themselves.

Reply:
I didn't understand